cmds/truncate/truncate.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Truncate - shrink or extend the size of a file to the specified size
+//
+// Synopsis:
+//     truncate [OPTIONS] [FILE]...
+//
+// Options:
+//     -s: size in bytes
+//     -c: do not create any files
+//
+// Author:
+//     Roland Kammerer <dev.rck@gmail.com>
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+)
+
+const cmd = "truncate [-c] -s size file..."
+
+var (
+	create  = flag.Bool("c", false, "Do not create files.")
+	sizeStr = flag.String("s", "", "Size in bytes, prefixes +/- are allowed")
```

Comment:
You can just use Int64, it allows +/-. In fact, flag.String is likely just a flag.Value that uses strconv.ParseInt just like you did.